x-pack/plugins/security_solution/common/endpoint/data_generators/exceptions_list_item_generator.ts

```diff
@@ -0,0 +1,50 @@
+/*
```

Comment:
a few new single-domain generators.

The ones for `fleet_*` below were cloned from the code in the `endpoint_data` generator. I will (in the near future - task 2262) refactor that one to use these internally instead of having its own implementation